Doppler.PushContact/Controllers/DomainController.cs

```diff
@@ -0,0 +1,29 @@
+using Doppler.PushContact.DopplerSecurity;
+using Doppler.PushContact.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace Doppler.PushContact.Controllers
+{
+    [Authorize(Policies.ONLY_SUPERUSER)]
+    [ApiController]
+    public class DomainController : ControllerBase
+    {
+        [HttpPut]
+        [Route("domains/{name}")]
+        public Task<IActionResult> Upsert([FromRoute] string name, [FromBody] bool isPushFeatureEnabled)
```

Comment:
If the PUT refers to the resource `domains/name`, I would expect that the body is a representation of the domain.

```suggestion
        public Task<IActionResult> Upsert([FromRoute] string name, [FromBody] Domain domain)
```

For example:

```diff
 ### Create/Update domain
 PUT /domains/pirulo.com
 Content-Type: application/json
 
-false
+{
+  "isPushFeatureEnabled": false
+}
  
 ### Get domain by name
 GET /domains/pirulo.com
 
 ## Success Response Body
 # {
 #   "name": "pirulo.com",
 #   "isPushFeatureEnabled": false
 # }
```

Reply:
It has sense! I see a minor situation. If we use `Domain` model as body content we have a redundant _domain name_ definition in out PUT method:
### Create/Update domain
 PUT /domains/**pirulo.com**
 Content-Type: application/json

{
   "name": "**pirulo.com**",
   "isPushFeatureEnabled": false
}

But It is OK, we must know that _body name_ definition will be not used

Reply:
Right, having a kind of field "read-only" that should be not required for POST or PUT verbs would be nice.

Reply:
Done! Now, we use `Domain` model as body content. In our examples files we do not include the `name` property in the body. Final solution:

> ### Create/Update domain
> PUT /domains/{name}
> Content-Type: application/json
> 
> {
>   "isPushFeatureEnabled": false
> }